django_orghierarchy/importers.py

```diff
@@ -110,7 +116,11 @@ def __init__(self, url, config=None):
 
     @property
     def fields(self):
-        return set(self.config['fields'])
```

Comment:
Remove the set operation as we are not doing membership lookup for the fields (we once were, but remove it after refactoring)